apps/judicial-system/web/src/routes/Defender/Cases/components/DefenderCasesTable.tsx

```diff
@@ -82,188 +42,117 @@ export const DefenderCasesTable: FC<Props> = ({
   } = useWithdrawAppealMenuOption()
 
   return (
-    <Box marginBottom={7}>
-      {loading ? (
-        <TableSkeleton />
-      ) : (
-        <table className={styles.table}>
-          <thead className={styles.thead}>
-            <tr>
-              <th className={styles.th}>
-                <Text as="span" fontWeight="regular">
-                  {formatMessage(tables.caseNumber)}
-                </Text>
-              </th>
-              <th className={cn(styles.th, styles.largeColumn)}>
-                <SortButton
-                  title={capitalize(
-                    formatMessage(core.defendant, { suffix: 'i' }),
-                  )}
-                  onClick={() => requestSort('defendants')}
-                  sortAsc={getClassNamesFor('defendants') === 'ascending'}
-                  sortDes={getClassNamesFor('defendants') === 'descending'}
-                  isActive={isActiveColumn('defendants')}
-                  dataTestid="accusedNameSortButton"
+    <>
+      <Table
+        thead={[
+          {
+            title: formatMessage(tables.caseNumber),
+          },
+          {
+            title: capitalize(formatMessage(core.defendant, { suffix: 'i' })),
+            sortable: { isSortable: true, key: 'defendants' },
+          },
+          {
+            title: formatMessage(tables.type),
+          },
+          {
+            title: formatMessage(tables.created),
+            sortable: { isSortable: true, key: 'created' },
+          },
+          { title: formatMessage(tables.state) },
+          {
+            title: formatMessage(
+              showingCompletedCases
+                ? tables.duration
+                : tables.hearingArrangementDate,
+            ),
+            sortable: showingCompletedCases
+              ? undefined
+              : { isSortable: true, key: 'courtDate' },
+          },
+        ]}
+        data={cases}
+        generateContextMenuItems={(row) => [
+          openCaseInNewTabMenuItem(row.id),
+          ...(shouldDisplayWithdrawAppealOption(row)
+            ? [withdrawAppealMenuOption(row.id)]
+            : []),
+        ]}
+        columns={[
+          {
+            cell: (row) => (
+              <CourtCaseNumber
+                courtCaseNumber={row.courtCaseNumber}
+                policeCaseNumbers={row.policeCaseNumbers}
+                appealCaseNumber={row.appealCaseNumber}
+              />
+            ),
+          },
+          {
+            cell: (row) => <DefendantInfo defendants={row.defendants} />,
+          },
+          {
+            cell: (row) => (
+              <ColumnCaseType
+                type={row.type}
+                decision={row.decision}
+                parentCaseId={row.parentCaseId}
+              />
+            ),
+          },
+          { cell: (row) => <CreatedDate created={row.created} /> },
+          {
+            cell: (row) => (
+              <Box display="flex" columnGap={1} rowGap={1} flexWrap="wrap">
```

Comment:
This CSS makes it so that the tags are placed side by side if there is enough space and wraps to two lines if not.

Reply:
Is it only relevant to include it for this table? or would it make sense to have it in the `TagCaseState` component to make it shared styling?

Reply:
You're right, this applies to other places as well. The `TagCaseState` component renders a single tag, so this doesn't apply there but I've created a new container component to be able to reuse this style elsewhere. Could you please take another look? 🙏